bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Indonesia.java

```diff
@@ -0,0 +1,550 @@
+/*
+ * BedWars1058 - A bed wars mini-game.
+ * Copyright (C) 2021 Andrei Dascălu
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Contact e-mail: andrew.dascalu@gmail.com
+ */
+
+package com.andrei1058.bedwars.language;
+
+import com.andrei1058.bedwars.BedWars;
+import com.andrei1058.bedwars.api.configuration.ConfigPath;
+import com.andrei1058.bedwars.api.language.Language;
+import com.andrei1058.bedwars.api.language.Messages;
+import org.bukkit.configuration.file.YamlConfiguration;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+
+import static com.andrei1058.bedwars.BedWars.mainCmd;
+
+public class Indonesia extends Language {
+
+    public Indonesia() {
+        super(BedWars.plugin, "id");
+
+        YamlConfiguration yml = getYml();
+        yml.options().copyDefaults(true);
+        yml.addDefault(Messages.PREFIX, "");
+        yml.addDefault("name", "Indonesia");
```

Comment:
this should be the name of Indonesian in your language. I'm waiting for a change.